SocketIOClientSwift/SocketIOClientOption.swift

```diff
@@ -68,6 +69,8 @@ public enum SocketIOClientOption: ClientOption {
             return .ReconnectAttempts(value as! Int)
         case "reconnectWait" where value is Int:
             return .ReconnectWait(value as! Int)
+        case "forceNew" where value is Bool:
```

Comment:
I'd recommend doing `switch (key, value)`, because then you can do `case ("forceNew", let value) where value is Bool)` and will automatically have the right type.